app/models/krikri/activity.rb

```diff
@@ -83,5 +83,23 @@ def rdf_subject
       RDF::URI(Krikri::Settings['marmotta']['ldp']) /
         Krikri::Settings['prov']['activity'] / id.to_s
     end
+
+    ##
+    # Return an Enumerator of URI strings of entities (e.g. aggregations or
+    # original records) that pertain to this activity
+    #
+    # @return [Enumerator] URI strings
+    def generated_entity_uris
```

Comment:
We are now working with both "generated" and "revised" entities.  I'm not sure whether this merits a change in these method signatures, but I thought it was worth mentioning.

I'd say we either keep these as is (with the excuse that PROV says that "Activities generate Entities"), or change them to just `#entity_uris` and `#entities`.  Thoughts?